comlink.ts

```diff
@@ -200,6 +200,7 @@ export const Comlink = (function() {
     activateEndpoint(endpoint);
     attachMessageHandler(endpoint, async function(event: MessageEvent) {
       if (!event.data.id) return;
+      if (!(event.data.msg && event.data.msg.callPath)) return;
```

Comment:
Where did you get the `msg` property from? `event.data` is a `InvocationRequest`. If you fix that, the tests should pass :)

Reply:
I got confused with how `MessageChannelAdapter` wraps the messages :|
My bad, fixing it right away.